Cython/Utility/TestCythonScope.pyx

```diff
@@ -40,6 +41,10 @@ cdef class TestClass(object):
     def def_cname_method(self, int value):
         print("Hello from def_cname_method", value)
 
+# For now cdef variables are outside the module scope,
+# so we can create a global cname that is easily accessible
+cdef PyObject* __pyx_TestClass_type = <PyObject*>TestClass
```

Comment:
I'm not quite sure I understand what this is actually testing, so I'm not sure if this is a valid workaround.

(Also, long-term I think these variables *should* be in the module state so it won't keep working. But that's a "later" problem)